Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -123,18 +155,6 @@ - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)thread
                         actionBlock:^{
                             [DebugUIMessages createFakeThreads:1000 withFakeMessages:1];
                         }],
-        [OWSTableItem itemWithTitle:@"🖼 fake media messages: 5"
```

Comment:
Once we've iterated on these changes a bit, we'll be able to remove many/most of the existing actions in this class.